lib/dialects/mysql/query.js

```diff
@@ -29,17 +29,17 @@ Query.prototype.run = function(sql, parameters) {
   //do we need benchmark for this query execution
   var benchmark = this.sequelize.options.benchmark || this.options.benchmark;
 
-  if (!benchmark) {
-    this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
-  } else {
+  if (benchmark) {
     var queryBegin = Date.now();
+  } else {
+    this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
   }
 
   var promise = new Utils.Promise(function(resolve, reject) {
     self.connection.query(self.sql, function(err, results) {
 
       if (benchmark) {
-        self.sequelize.log('Executed (' + (self.connection.uuid || 'default') + '): ' + self.sql + ' in ' + (Date.now() - queryBegin) + 'ms', self.options);
+        self.sequelize.log('Executed (' + (self.connection.uuid || 'default') + '): ' + self.sql, (Date.now() - queryBegin), self.options);
```

Comment:
Apparently we don't have tests for it, but if a user is expecting options to be in the second argument you just broke his code.

4.0 is months away, i'd suggest you rather rewrite this to have the benchmark be part of the options passed to the  log call.